schema.graphql

```diff
@@ -1,3 +1,5 @@
 type PureProxy @entity {
   id: ID! # pure proxied accountId
+  blockNumber: String
```

Comment:
I would suggest to use:
```suggestion
  blockNumber: Int! @index
```

also, I faced that request for ID is working slow, Is it reasonable to add @index for id too?